projects/step-frontend/src/lib/modules/timeseries/chart/time-series-chart.component.ts

```diff
@@ -234,20 +251,20 @@ export class TimeSeriesChartComponent implements OnInit, AfterViewInit, OnChange
    * Switches the state of the specified series.
    */
   toggleSeries(id: string): void {
-    const index = this.seriesIndexesByIds[id];
```

Comment:
I dont see these variables being reassigned anywhere, however this is the sort of thing that falls into the personal preference category, but I would still argue that this only contributes to more changes without providing actual value

Reply:
agree